yarn.lock

```diff
@@ -15,6 +15,441 @@
     "@jridgewell/gen-mapping" "^0.3.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
+"@aws-crypto/crc32@3.0.0":
```

Comment:
Are all of this packages required? This is a lot....

Reply:
Removed them. It is supposed to be in my new plugin not here.